src/app/pages/landpage/landpage.component.ts

```diff
@@ -1,16 +1,46 @@
 import { Component, OnInit } from '@angular/core';
+import {IAuthData} from '../../../shared/interfaces';
+import {Router} from '@angular/router';
+import {AuthService} from '../../../services/authservice';
+import {UserService} from '../../../services/userservice';
 
 @Component({
   selector: 'app-landpage',
   templateUrl: './landpage.component.html',
   styleUrls: ['./landpage.component.css']
 })
-export class LandpageComponent implements OnInit {
+export class LandpageComponent {
   public showRegist = false;
-
-  constructor() { }
-
-  public ngOnInit() {
+  public user: IAuthData = {email: '', password: ''};
+  constructor(
+    private router: Router,
+    private authservice: AuthService,
+    public _user: UserService
+  ) {}
+  // Calling Exit function from AuthService
+  quit() {
+    this.authservice.logOutFunk();
+  }
+  // Calling boolean status variable logged from AuthService
+  lLogged() {
```

Comment:
Add "public" in front of function declaration.
FUNCTION NAMING!!! "lLogged" - what does that mean?